test/libsolidity/syntaxTests/inlineAssembly/invalid/variable_declaration_suffix_offset.sol

```diff
@@ -1,15 +1,11 @@
 contract C {
     function f() public pure {
         assembly {
-            let x_offset := 1
-            let x_slot := 1
-            let _offset := 1
-            let _slot := 1
+            let x.offset := 1
+            let x.slot := 1
         }
     }
 }
 // ----
-// DeclarationError 9155: (79-87): In variable declarations _slot and _offset can not be used as a suffix.
-// DeclarationError 9155: (109-115): In variable declarations _slot and _offset can not be used as a suffix.
-// DeclarationError 9155: (137-144): In variable declarations _slot and _offset can not be used as a suffix.
-// DeclarationError 9155: (166-171): In variable declarations _slot and _offset can not be used as a suffix.
+// DeclarationError 3927: (79-87): User-defined identifiers in inline assembly cannot contain '.'.
```

Comment:
Hmm, wouldn't the better error here be saying `x` is not found?

Reply:
ping @chriseth

Reply:
If this were `x.offset := 1`, then yes, but how would you declare a new yul variable overwriting an existing solidity variable?